### PR TITLE
Expose Development.IDE.Core.Preprocessor

### PR DESCRIPTION
**Pull request by tittoassini**
Preprocessor functionality is required by the Eval Plugin.

### CODE REVIEW

File: src/Development/IDE/GHC/Orphans.hs [hunk 1]
```diff
@@ -10,14 +10,14 @@
...
 
-import GHC
-import GhcPlugins
-import Development.IDE.GHC.Compat
-import qualified StringBuffer as SB
-import Control.DeepSeq
-import Data.Hashable
-import Development.IDE.GHC.Util
-import Bag
+import           Bag
```
Comment (ndmitchell) [inline src/Development/IDE/GHC/Orphans.hs:13]: As a project we don't like this style - better not to indent things more than necessary.

**Review by ndmitchell — COMMENTED** [review]
This seems a perfectly reasonable thing to expose - its pretty well designed as an API.

File: src/Development/IDE/GHC/Orphans.hs (hunk 1 above)
Comment (tittoassini) [reply src/Development/IDE/GHC/Orphans.hs:13]: I seem to have made a mistake, the first two commits have already been accepted as part of a previous PR.

The one I meant to submit is only the last one (exposing Development.IDE.Core.Preprocessor).

Comment (tittoassini) [reply src/Development/IDE/GHC/Orphans.hs:13]: Sorry for the confusion, I have opened a different PR with just the required change.

**Pull request closed without merging**